README.md

```diff
@@ -112,16 +112,19 @@ the TurtleBot itself or using a cross-build solution, like the one provided with
 Before you build the robot workspace, you must edit the file `robot_ws/src/object_tracker_robot/config/model_config.yaml` to include the location
 of your trained model.
 
-Create a file `/etc/ros/rosdep/custom-rules/object-tracker-rules.yaml` and add the following configuration to it.
+Next, you need to add custom rosdep rules required by the application. The first rule is hosted by RoboMaker, and can be added by running `echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/python.yaml" > /etc/ros/rosdep/sources.list.d/18-python-boto3-pip.list`
+
+The next rule you can create yourself. Create a file `/etc/ros/rosdep/custom-rules/object-tracker-rules.yaml` and add the following configuration to it.
 ```bash
 libjpeg62:
   ubuntu:
     xenial: [libjpeg62]
 ```
 
-Then add the rule to rosdep using the following commands:
+Then add the rule to rosdep bu running `echo "yaml file:/etc/ros/rosdep/custom-rules/object-tracker-rules.yaml" > /etc/ros/rosdep/sources.list.d/23-object-tracker-rules.list`
```

Comment:
This seems very odd to me that the previous command creates a file with some configuration in it for rosdep and then on this line you add that file as a reference for this other file. Did `23-object-tracker-rules.list` already exist before running this command? If so, where did it come from? 

Why can't the configuration from lines 119-121 just b e added directly to `23-object-tracker-rules.list` instead of being a separate file? 

If both of these files are already in the `/etc/ros/rosdep/custom-rules` folder then why does one need to reference the other? Shouldn't they both be picked up anyways? 

Also, there is a minor typo, `bu` should be `by`.

Reply:
The first file, `/etc/ros/rosdep/custom-rules/object-tracker-rules.yaml`, contains the rosdep rule, which can be in any location. When rosdep updates its rule list, it reads the files located in `/etc/ros/rosdep/sources.list.d/` and then looks for rules in the locations defined in those files. The second file, `/etc/ros/rosdep/sources.list.d/23-object-tracker-rules.list`, defines the location of the first file so that rosdep can find it when it updates.

The command on line 124 will create a new file. If `23-object-tracker-rules.list` already exists, then it will be overwritten. However, given that the name of the file is fairly specific to the sample application, it's unlikely that the file exists unless it was created by a previous run of the sample application.

Reply:
The typo is fixed in a separate commit.

Reply:
Thanks for the explanation. Seems like there should be a way to just directly include a rule in a file inside the `sources.list.d` directory instead of requiring an intermediate file.